portal-web/docroot/html/taglib/ui/search_iterator/list.jsp

```diff
@@ -246,7 +248,6 @@ if (fixedHeader) {
 
 						textSearchEntry.setAlign(rowChecker.getAlign());
 						textSearchEntry.setColspan(rowChecker.getColspan());
-						textSearchEntry.setCssClass(rowChecker.getCssClass());
```

Comment:
What was the reason for removing this? Did you see something broken that removing this fixed?

Reply:
When I added `rowChecker.setCssClass("custom-control-input");` on line 238, the class name `custom-control-input` gets added to the `td` element and the `input` element. I tried applying `rowChecker.setCssClass("custom-control-input");` after `textSearchEntry`, but the class didn't get applied to the input element 🤷. I'm not sure why.

Reply:
Okay, @pat270  go ahead and send this to FI for review, explaining that and they should be able to tell you if it's okay or not.